File: plugin/src/test/java/com/oldstats/OldStatsPluginTest.java
package com.oldstats;

import net.runelite.client.RuneLite;
import net.runelite.client.externalplugins.ExternalPluginManager;

/**
 * Launches a real RuneLite client with {@link OldStatsPlugin} preloaded, so it
 * can be tried against a live game without publishing to the Plugin Hub — the
 * standard pattern for developing a RuneLite plugin outside the main RuneLite
 * source tree. Run via {@code ./gradlew runClient} (see build.gradle.kts), or
 * directly from an IDE by running this class's {@code main}.
 */
public class OldStatsPluginTest {
    public static void main(String[] args) throws Exception {
        ExternalPluginManager.loadBuiltin(OldStatsPlugin.class);
        RuneLite.main(args);
    }
}

File: plugin/src/test/kotlin/com/oldstats/OldStatsPluginTest.kt
package com.oldstats

import net.runelite.client.RuneLite
import net.runelite.client.externalplugins.ExternalPluginManager

/**
 * Launches a real RuneLite client with [OldStatsPlugin] preloaded, so it can
 * be tried against a live game without publishing to the Plugin Hub — the
 * standard pattern for developing a RuneLite plugin outside the main
 * RuneLite source tree. Run via `./gradlew runClient` (see build.gradle.kts),
 * or directly from an IDE by running this file's `main`.
 */
fun main(args: Array<String>) {
    ExternalPluginManager.loadBuiltin(OldStatsPlugin::class.java)
    RuneLite.main(args)
}
